dao-canvas-template.md

```diff
@@ -0,0 +1,286 @@
+- Process/framework for decentralised decision making based on our mission/values and goals.
+
+Future:
+
+- Value capture and fair distribution of ownership
+- Growth of a treasury/funding to support community projects
+
+I've been picturing it like Bankless (other DAO's seasons). Something along the lines of the following:
+
+**Season 0**
+
+Season 0 (to end of October/November?)
+
+- Finalizing our mission statement/manifesto + publishing this
+- Coms/decision making flow and where each happens)
+- Org structure (team, roles)
+- How do we plan to recognise and reward contributions
+- Branding and V1 website (about, on-site minting, docs?, mission statement)
+- Token allocation and distribution
+- Focus for season 1
+
+**Season 1** (Decided by community)
+
+## Community
+
+## Project backstory
+
+The project was started as an experiment by [Nader Dabit](https://twitter.com/dabit3) to see what can be achieved when we bring together talented and like-minded people who are interested in learning about and building on Web3.
+
+Developer DAO draws inpiration from other creator-focused DAOs like [Friends with Benefits](https://twitter.com/fwbtweets), 
+
+Since launch, over 3000 NFT have been sold and over 900 of them have joined the discord server.
+
+## Communication platforms
+
+- **Conversations/Brainstorms:** _Discord_
+- **conform discussion/Proposals/Consensus building:** _Discourse_
+- **Knowledge repository:** _GitHub, developerdao.com / docs, Notion?, newsletter, Blog (multiple for different content types?), YouTube/similar_
+
+### Communication rhythms
+
+- **Weekly:** project/team/guild calls, Community call, Core team meeting
+- **Monthly:** Town Hall
+- **Quartely:** Season round-up / launch (replaces months town hall, potentially held via twitter spaces
+- )
+
+### Who is responsible?
+
+- **Core team:**
+
+The closest thing we have currently have to full-time employees. Responsible for executing the mission of the DAO on a day-to-day basis/week-by-week basis (communication, organisations, docs, shepparding governance processes). Must accept minimum commitment to ensure smooth running of DAO (i.e regular meetings, responsibility for specific area/domain)
+
+- **Team/Guilds/Project leads:**
+
+TBD
+
+- **Contributors/Resource pool:**
+
+Members actively invovled in contributing on an ad-hoc basis.
+
+_NEEDS THOUGHT_
+
+- **Community members:**
+
+The wider, more passive community. Critical for helping the DAO and grow and easy upwards mobiltiy through to other roles (above)
+
+## Member journey
+
+### Describe the new user on-boarding process
+
+To start, the sole way of joining the community is by minting one of the D4R genesis NFTs. Any holder can then access our private discourse server and begin a simple on-boarding process to assign themselves a skill role.
+
+We acknowledge this could be more inclusive given the associated gas fee however we believe it is best solution as this time to ensure members have "skin-in-the-game".
```

Comment:
```suggestion
We acknowledge this could be more inclusive given the associated gas fee, however we believe it is best solution at this time to ensure members have "skin-in-the-game".
```